src/components/fields/ArrayField.js

```diff
@@ -12,8 +12,7 @@ import {
   retrieveSchema,
   toIdSchema,
   shouldRender,
-  getDefaultRegistry,
-  setState
```

Comment:
We should check that this is still needed elsewhere. If not, we should remove that helper function, which is now obsolete.